packages/core/src/providers/studio/index.ts

```diff
@@ -120,74 +124,89 @@ export class StudioLivepeerProvider extends BaseLivepeerProvider {
     return studioStreamSessions;
   }
 
-  async createAsset(args: CreateAssetArgs): Promise<Asset> {
-    if (args.url) {
```

Comment:
This functionality shouldn't have been removed - this was a new feature that we'll need to add back

Reply:
This still needs to be added back